instrumentor/controllers/deleteinstrumentedapplication/common.go

```diff
@@ -57,40 +56,19 @@ func deleteWorkloadInstrumentedApplication(ctx context.Context, kubeClient clien
 		return client.IgnoreNotFound(err)
 	}
 
-	logger := log.FromContext(ctx)
-	logger.V(1).Info("instrumented application deleted", "namespace", ns, "name", name, "kind", kind)
-	return nil
-}
-
-func isWorkloadInstrumentationEffectiveEnabled(ctx context.Context, kubeClient client.Client, obj client.Object) (bool, error) {
-
-	// if the object itself is labeled, we will use that value
-	workloadLabels := obj.GetLabels()
-	if val, exists := workloadLabels[consts.OdigosInstrumentationLabel]; exists {
-		return val == consts.InstrumentationEnabled, nil
-	}
-
-	// we will get here if the workload instrumentation label is not set.
-	// no label means inherit the instrumentation value from namespace.
-	var ns corev1.Namespace
-	err := kubeClient.Get(ctx, client.ObjectKey{Name: obj.GetNamespace()}, &ns)
+	err = kubeClient.Delete(ctx, &odigosv1.InstrumentationConfig{
```

Comment:
above we return if delete on `InstrumentedApplication` failed. I think we should make sure to also attempt to delete the `InstrumentationConfig` even if `InstrumentedApplication` is not found